src/App/GeoFeatureGroupExtension.cpp

```diff
@@ -407,7 +407,7 @@ bool GeoFeatureGroupExtension::extensionGetSubObject(DocumentObject*& ret,
             *mat *= const_cast<GeoFeatureGroupExtension*>(this)->placement().getValue().toMatrix();
         }
     }
-    else if ((dot = strchr(subname, '.'))) {
+    else if (dot = strchr(subname, '.')) {
```

Comment:
This is usually the way to suppress warnings when assigning variables in an if. Have we removed that type of warning?

Reply:
MSVC doesn't warn, but clang does. I'll drop this change.